cmd/satellite/gracefulexit.go

```diff
@@ -98,3 +104,46 @@ func generateGracefulExitCSV(ctx context.Context, completed bool, start time.Tim
 	}
 	return err
 }
+
+func verifyGracefulExitReceipt(ctx context.Context, identity *identity.FullIdentity, receipt string) error {
```

Comment:
Maybe we should also have an input for the requested storage node id so we can check that the receipt is actually for that storage node?

Reply:
Added ID checks